spec/api_client_spec.rb

```diff
@@ -10,10 +10,10 @@ module Api
       describe '#append_to_stream' do
 
         it "should handle one event" do
-          event = Event.new('event_type', { data: 1 })
```

Comment:
You are adding new thing here - so why changing existing tests instead of just adding a new one ?